### DISCUSSION TITLE
Is Helm support still truly experimental?

### DISCUSSION THREAD

**Discussion by jemag** [grafana/tanka#811] [General]
We are currently considering using Tanka heavily with ArgoCD to manage a lot of our applications. Our 2 main use cases are : 
- Using tanka for the stuff that we developed ourselves
- Using tanka to modify 3rd party helm charts (currently this is done through using kustomize on top of helm template, which I am not the biggest fan of)

It seems that the Helm support was released around August 2020, but the documentation still mentions this features as `Experimental` [(doc)](https://tanka.dev/helm). Given that the feature has been out for more than 2 years now, what we are wondering is:
**1. Is Helm support truly still experimental?
2. How risky would it be to use this feature relatively extensively for our GitOps setup?**

**Comment by Duologic** [comment]
Good question.

Internally we've been leveraging Helm support all over the place, the feature is quite stable and hasn't undergone big changes recently. We are still figuring out a good automated upgrade path for charts, it's quite manual right now. That being said, I'd say it would be safe to remove the experimental warning. Big changes would require a migration path for ourselves that it would be included in Tanka anyway.

**Reply by jemag** [reply]
Thanks for this, it is nice to know that it is used quite a bit internally, and reassuring for new comers looking into using it as well.

**Comment by frimik** [comment]
Bumping helm charts is something I’ve thought about too but haven’t had time yet. 

I’m using `vendir` for all chart vendoring and the TODO I have in my mind is to utilize renovatebot with regex manager or rely on luck that someone gets to this issue before I get to automating this; https://github.com/renovatebot/renovate/issues/17711
